src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+
+public class HomePage extends BasePage{
+    public HomePage (WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By home = By.cssSelector(".home");
+    By currentQueue = By.cssSelector(".queue.active");
+    By allSongs = By.cssSelector(".songs.active");
+    By albums = By.cssSelector(".albums.active");
+    By artists = By.cssSelector(".artists.active");
+    By favorites = By.cssSelector(".playlist.favorites");
+    By recentlyPlayed = By.cssSelector(".playlist.recently-played");
+    By userProfile = By.cssSelector("[title='iew/edit user profile']");
+    By aboutKoel = By.cssSelector("#searchForm input");
+    By createNewPlaylist = By.cssSelector("[data-testid='sidebar-create-playlist-btn']");
+    By newPlaylistContextMenu = By.cssSelector("[data-testid='playlist-context-menu-create-simple']");
+    By playlistNameInputField = By.cssSelector("[name='create-simple-playlist-form'] input");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```